nextflow.config

```diff
@@ -14,6 +14,7 @@ params {
 
     // References - Not used in funcscan, left for template purposes
     genome                     = null
+    fasta                      = null
```

Comment:
```suggestion
```
A mistake in the template. See discussion in #pipeline-maintainers: https://nfcore.slack.com/archives/C04QR0T3G3H/p1709811356971549